components/header/podIndicator/styles.js

```diff
@@ -38,7 +38,10 @@ const styles = (theme) =>
     },
     popover: {
       padding: theme.spacing(2, 2, 0),
-      width: 450,
+      maxWidth: "100vw",
```

Comment:
I'm don't think the `100vw` is important here, e.g. you could've done `initial` as well. That said, it does make it so that the input field is not wider than the screen available, so I'm ok with this solution.